desci-server/src/theGraph.ts

```diff
@@ -94,15 +94,16 @@ export const getIndexedResearchObjects = async (
     logger.info({ streamHistory }, "Resolver history for nodes found");
   };
 
-  // 2. These are simplest to fallback to graph lookup as we don't have the dPID handy
-  const nodesWithoutStream = nodeRes
-    .filter(n => !n.ceramicStream)
-    .map(n => n.uuid);
```

Comment:
For example, `dpid/1` (UUID `Mm03P0gZc75jv03U6AaMePvDIaweZ-2cEUdOfhS__Co`) doesn't exist in the dev database. Since this previously filtered the results from Prisma, all UUIDs in `urlSafeBase64s` that isn't loaded into the database were getting dropped between the cushions here, filtered out before being sent to the graph lookup.